flask_resty/api.py

```diff
@@ -171,12 +163,3 @@ def ping():
 class FlaskRestyState(object):
```

Comment:
we could remove this entirely and make the registered extension just be the `Api` object, but this is probably more future-proof